doc/libpmem/pmem_is_pmem.3.md

```diff
@@ -75,15 +75,16 @@ responsibility for flushing stores to persistence will find the
 functions described in this section to be the most commonly used.
 
 The **pmem_is_pmem**() function detects if the entire range
-\[*addr*, *addr*+*len*) consists of persistent memory.
-The implementation of **pmem_is_pmem**() requires a non-trivial amount
-of work to determine if the given range is entirely persistent memory.
-For this reason, it is better to call **pmem_is_pmem**() once when a
-range of memory is first encountered, save the result, and use the saved
-result to determine whether **pmem_persist**(3) or **msync**(2) is
-appropriate for flushing changes to persistence. Calling
-**pmem_is_pmem**() each time changes are flushed to persistence will
-not perform well.
+\[*addr*, *addr*+*len*) consists of persistent memory. To detect
```

Comment:
repetition of 'detect'.

`Calling this function with a memory range that originates from source different than **pmem_map_file()** is undefined.`

or

`The result is undefined if called on a range that does not originate in its entirety from **pmem_map_file()**.`


Or something like that.